.github/workflows/blackbox-main.yml

```diff
@@ -42,6 +42,18 @@ jobs:
       - name: Prepare
         uses: ./.github/actions/prepare
 
+      - name: Prune Dependencies
+        shell: bash
```

Comment:
Just a detail, but you don't need to specify `shell: bash` as this is the default anyway: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Reply:
TIL, will give it a try. Thanks! 😄
Adding a mental note to also remove it from the prepare action.
 https://github.com/directus/directus/blob/16e625cdd1ff756a21c43ce077c266042a120cf5/.github/actions/prepare/action.yml#L38-L40

Reply:
Apparently for the prepare action, the `shell` field is mandatory according to the linter.

<img width="355" alt="image" src="https://user-images.githubusercontent.com/26413686/199502771-267cbd6f-417f-4507-a18c-f9b0eb9bb0ae.png">

Reply:
Thanks 😃
In the case of actions, however, it is somewhat different: Since you cannot define the runner there (action could also run on Windows) you HAVE to specify the shell 😄